packages/forms/src/UIForm/fields/TextArea/displayMode/TextMode.component.js

```diff
@@ -0,0 +1,29 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import { TextMode as FieldTemplate } from '../../FieldTemplate';
+
+export default function TextMode({ id, schema, value }) {
+	const { rows = 5, title } = schema;
+
+	return (
+		<FieldTemplate id={id} label={title}>
+			<pre style={{ height: `${rows * 2}rem`, fontSize: 'inherit' }}>{value}</pre>
```

Comment:
do we have to make inline style here ?

Reply:
There is a feature in Textarea about the size. We can pass a row attributes. I just wanted to make the div `almost` have the same size as the textarea.